tcms/testruns/static/testruns/js/get.js

```diff
@@ -86,9 +84,50 @@ function renderProgressBars(positiveCount, negativeCount, allCount) {
 
 function renderCountPerStatusList(statusCount) {
     for (var status in statusCount) {
-        const status_id = statusCount[status].id;
+        const statusId = statusCount[status].id;
 
-        $(`#count-for-status-${status_id}`).attr('href', `?status_id=${status_id}`);
-        $(`#count-for-status-${status_id}`).text(statusCount[status].count);
+        $(`#count-for-status-${statusId}`).attr('href', `?status_id=${statusId}`).text(statusCount[status].count);
     }
 }
+
+function renderTestExecutions(testRunId, executionStatuses) {
+    const container = $('#test-executions-container')
+    const testExecutionRowTemplate = $('#test-execution-row')[0].content
+
+    jsonRPC('TestExecution.filter', { 'run_id': testRunId }, testExecutions => {
+        testExecutions.forEach(testExecution => {
+            const executionStatus = executionStatuses.find(status => status.id === testExecution.status_id)
+
+            jsonRPC('TestCase.filter', { 'id': testExecution.case_id }, testCases => {
+                jsonRPC('Bug.filter', {'execution_id': testExecutions.id}, bugs => {
+                    const template = $(testExecutionRowTemplate.cloneNode(true))
+
+                    container.append(renderTestExecutionRow(template, testExecution, testCases[0], executionStatus, bugs))
+                }, true)
+            }, true)
+        })
+
+        treeViewBind();
+    })
+
+}
+
+function renderTestExecutionRow(template, testExecution, testCase, testExecutionStatus, bugs) {
+    template.find('.test-execution-info').html(`TE-${testExecution.id}`)
+    template.find('.test-execution-info-link').html(testExecution.case)
+    template.find('.test-execution-info-link').attr('href', `/case/${testExecution.case_id}`)
```

Comment:
Missing trailing `/` which will cause Django to make a redirect because it likes it's trailing slashes.